smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -235,7 +235,6 @@ contract MixinTransfer is
     uint _time
   )
     public view
-    hasValidKey(_keyOwner)
```

Comment:
@NickCuso 
yup, it overflows. 2 ways I can handle this.
1.) Restore the original `require` statement
2.) Use safe math to guard against the overflow

Generally, I would favour #1, but #2 is more in line with the purpose of the issue this PR is addressing.

Reply:
hmm i think i'd vote for 1. safe math would also cause a revert but the one we had is more clear

Reply:
thinking on this a bit, do you think it might be better to return 0 for this scenario? i'm not sure yet

Reply:
so something like:
```
if(! getHasValidKey(_keyOwner) {
  return 0;
}
```

Reply:
@NickCuso It might may more sense in this context, and would be a bit simpler to integrate with. The caller is expecting a `uint`, and would get one consistently.